cmake/app/boilerplate.cmake

```diff
@@ -191,7 +191,12 @@ set(CACHED_BOARD ${BOARD} CACHE STRING "Selected board")
 # e.g. zephyr/boards/arm/96b_carbon_nrf51/96b_carbon_nrf51_defconfig. When
 # found, use that path to infer the ARCH we are building for.
 if(NOT BOARD_ROOT)
-  set(BOARD_ROOT ${ZEPHYR_BASE})
+  if(DEFINED ENV{BOARD_ROOT})
+    set(BOARD_ROOT $ENV{BOARD_ROOT})
+    message(STATUS "BOARD_ROOT set from the environment variable with the same name to ${BOARD_ROOT}")
```

Comment:
BOARD_ROOT's verbosity must match board's. So there should be no message here.

Reply:
Roger, it was just to avoid new mysterious things from happening

Reply:
Now that there are multiple sources that can set BOARD_ROOT, a message like this might be in order:

message(STATUS "Selected BOARD_ROOT ${BOARD_ROOT}")

to match BOARD.